.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+
+jdk: oraclejdk8
+
+before_script:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
I have a doubt, `-DskipTests` command's task is done by `-Dmaven.javadoc.skip=true`, doesn't it? 
see this https://www.mkyong.com/maven/how-to-skip-maven-unit-test/